src/modules/Topics/components/TopicAdvanceConfig/CoreConfiguration.tsx

```diff
@@ -294,15 +292,17 @@ const CoreConfiguration: React.FC<CoreConfigurationProps> = ({
         fieldId='replicas'
         btnAriaLabel={t('topic.replicas')}
         fieldLabel={t('topic.replicas')}
-        fieldValue={isCreate ? DEFAULT_REPLICAS : topicData.replicationFactor}
+        fieldValue={
+          isCreate ? replicationFactor?.toString() : topicData.replicationFactor
+        }
         popoverBody={t('topic.replicas_description')}
         popoverHeader={t('topic.replicas')}
       />
       <TextWithLabelPopover
         fieldId='min-insync-replicas'
         btnAriaLabel='topic detail min-in-sync replica'
         fieldLabel='Minimum in-sync replicas'
-        fieldValue={DEFAULT_MIN_INSYNC_REPLICAS}
+        fieldValue={minInSyncReplicas?.toString()}
```

Comment:
This will show "undefined" when field is not there. I think we need to define values once to control what will be displayed. `undefined` should never appear as value